inst/schema/QueueStatus.schema.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "QueueStatus",
+    "type": "array",
```

Comment:
Can you structure this output so that we have

```json
{
  "tasks": [
      {
          "key": "antiutopian_peregrinefalcon",
          "status": "running",
          "name": "minimal"
      },
      {
          "key": "flavoured_bassethound",
          "status": "queued",
          "name": "other"
      }
  ]
}
```

or similar. Then we can, here or elsewhere, add other information such as the number of workers.

for the running job do you want to return the orderly id too?

Reply:
Thanks yeah, sounds good. Made these changes now